### PR TITLE
[5.3] Add model methods only and except

### PR DESCRIPTION
**Pull request by ntzm**
Feels like `except` might be a bit inefficient because it will load relations even if they are "ignored". Any ideas on how to improve it without replicating `attributesToArray`?

### CODE REVIEW

File: src/Illuminate/Database/Eloquent/Model.php [hunk 1]
```diff
@@ -3067,6 +3067,44 @@ public function is(Model $model)
...
+    public function only($keys = [])
+    {
+        $keys = is_array($keys) ? $keys : func_get_args();
+
+        $result = [];
+
+        foreach ($keys as $key) {
+            $value = $this->getAttribute($key);
+
+            if (! is_null($value)) {
```
Comment (taylorotwell) [inline src/Illuminate/Database/Eloquent/Model.php]: This might be a little problematic if the database value is actually `null`.

Comment (ntzm) [inline src/Illuminate/Database/Eloquent/Model.php]: Good point. Don't know why I did that. Will update

**Comment by taylorotwell** [comment]
Going to hold off on this since I feel like you could just do Arr::only($model->toArray()... manually.

**Comment by ntzm** [comment]
Fair enough! Thanks for taking the time to review

**Pull request closed without merging**